src/tests/yararules/test_corpus_yara_rule.py

```diff
@@ -14,13 +14,13 @@
 class TestYaraRules(unittest.TestCase):
     def test_regression(self):
 
-        yara_files = [f for f in os.listdir(testdir) if ".txt" not in f]
+        yara_files = [f for f in os.listdir(testdir) if f.endswith(".yar")]
```

Comment:
We can actually use pathlib here

```suggestion
        yara_files = Path(testdir).glob("*.yar")
```